shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/MealApi.kt

```diff
@@ -15,25 +16,30 @@ class MealApi {
             val json = Json { ignoreUnknownKeys = true }
             serializer = KotlinxSerializer(json)
             defaultRequest {
-                contentType(ContentType.Application.Json)
```

Comment:
Was still having some issues with this as a default request 😖

Reply:
maybe we should leave the default request out then for what it's worth

Reply:
Yeah, we send the URL in each call anyway - I will get rid of it and check it still works as expected